software/openvisualizer/bin/openVisualizerApp/SConscript

```diff
@@ -49,7 +49,10 @@ def uiRunner(target, source, env):
     
     if env['SIMTOPOLOGY']:
         argList.append('--simTopology={0}'.format(env['SIMTOPOLOGY']))
-    
+   
+    if env['PATHTOPO']:
```

Comment:
please use the same name for your env var and command line swicth, i.e. `PATHTOPO` and `pathTopo`